### PR TITLE
Add metrics port for modelmesh infra pods

### PR DESCRIPTION
**Pull request by rimolive**
Signed-off-by: Ricardo Martinelli de Oliveira <rmartine@redhat.com>

## Description
Add metrics port for modelmesh services

## How Has This Been Tested?
* Deploy modelmesh kfdef
* Configure prometheus to scrape metrics from the following service objects:
    * odh-model-controller-metrics-service
    * modelmesh-controlle
* Check if the targets are available

## Merge criteria:

- [X] The commits are squashed in a cohesive manner and have meaningful messages.
- [X] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [X] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: model-mesh/manager/manager.yaml [hunk 1]
```diff
@@ -84,3 +84,18 @@ spec:
...
+    control-plane: modelmesh-controller
+  name: modelmesh-controller
+spec:
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 8080
+  type: ClusterIP
+  selector:
+    control-plane: modelmesh-controller
```
Comment (anishasthana) [inline model-mesh/manager/manager.yaml]: Missing newline

**Review by anishasthana — APPROVED** [review]

**Comment by anishasthana** [comment]
This is actually a little premature. /hold

**Comment by anishasthana** [comment]
We will be raising a PR later for a bulk update of the manifests related to modelmesh-serving

**Review by anishasthana — CHANGES_REQUESTED** [review]
This PR needs to be closed out.

**Comment by LaVLaS** [comment]
Closing based on the recommendation of @anishasthana

**Pull request closed without merging**